scripts/traas.sh

```diff
@@ -63,13 +67,13 @@ for PROJFULLREF in $ZUUL_REFS ; do
     project=${change[0]}
     branch=${change[1]}
     ref=${change[2]}
-    if [ "$project" = "openstack-infra/tripleo-ci" ]; then
+    if [ "$project" = "openstack-infra/tripleo-ci" -a ! -d tripleo-ci ]; then
```

Comment:
this is fine. i guess I handled it below by rm'ing first. so it depends on the behavior you want.